app/services/flowmanagement/pagerouters/removeflow/UpdateAnyInformationPageRouter.scala

```diff
@@ -29,19 +29,21 @@ import uk.gov.hmrc.play.frontend.auth.AuthContext
 
 import scala.concurrent.{ExecutionContext, Future}
 
-@Singleton
-class UpdateAnyInformationPageRouter @Inject()(val statusService: StatusService,
-                                               val businessMatchingService: BusinessMatchingService) extends PageRouter[RemoveBusinessTypeFlowModel] {
-
-  override def getPageRoute(model: RemoveBusinessTypeFlowModel, edit: Boolean = false)
-                           (implicit ac: AuthContext,
-                            hc: HeaderCarrier,
-                            ec: ExecutionContext
 
-                           ): Future[Result] = {
-    ???
-  }
-}
+//TODO  IS this file redundant
+//@Singleton
+//class UpdateAnyInformationPageRouter @Inject()(val statusService: StatusService,
+//                                               val businessMatchingService: BusinessMatchingService) extends PageRouter[RemoveBusinessTypeFlowModel] {
+//
+//  override def getPageRoute(model: RemoveBusinessTypeFlowModel, edit: Boolean = false)
+//                           (implicit ac: AuthContext,
+//                            hc: HeaderCarrier,
+//                            ec: ExecutionContext
+//
+//                           ): Future[Result] = {
+//    Future.successful(Redirect(removeRoutes.NeedToUpdateController.get()))
+//  }
+//}
```

Comment:
Can we remove this if it's no longer needed?

Reply:
Removed